src/current/v23.2/drop-virtual-cluster.md

```diff
@@ -0,0 +1,70 @@
+---
+title: DROP VIRTUAL CLUSTER
+summary: The DROP VIRTUAL CLUSTER statement deletes a virtual cluster.
+toc: true
+docs_area: reference.sql
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include enterprise-feature.md %}
+
+{% include_cached new-in.html version="v23.2" %} The `DROP VIRTUAL CLUSTER` statement removes virtual clusters. Virtual clusters are only used as part of the [physical cluster replication]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}) workflow.
+
```

Comment:
We should warn people that this will delete any data related to this virtual cluster and cannot be undone.

Reply:
I have added a warning to this effect!